docs/src/tutorial.md

```diff
@@ -69,3 +69,67 @@ p.options.max_bb_eval = 1000
 result = solve(p, [0.0;2.0])
 println("Solution: ", result.x_best_feas)
 ```
+
+## Linear equality constraints
+
+It is equally possible to define linear equality constraints on a NomadProblem. In this case,
+NOMAD will solve the original problem on a reduced subspace of the original problem, via
+a conversion. For more details, the reader is invited to refer to:
+
+C. Audet, S. Le Digabel and M. Peyrega, Linear equalities in blackbox optimization.
+*Computational Optimization and Applications*, 61(1), 1-23, May 2015.
+
+We consider the following example:
+
+```math
+\begin{array}{rl}
+  (BB) \ \ \
+  \displaystyle \min_{x \in \mathbb{R}^5} & (x_1 -1)^2 + (x_2 - x_3)^2 + (x_4 - x_5)^2\\
+  s.t. & x_1 + x_2 + x_3 + x_4 + x_5 = 5 \\
+  & x_3 -2 x_4 - 2 x_5 = -3\\
+  & -10 \leq x_1, x_2, x_3, x_4, x_5 \leq 10\\
+\end{array}
+```
+
+This problem can be solved by Nomad the following way:
+```@example HS48
+
+# blackbox
+function bb(x)
+  f = (x[1]- 1)^2 * (x[2] - x[3])^2 + (x[4] - x[5])^2
+  bb_outputs = [f]
+  success = true
+  count_eval = true
+  return (success, count_eval, bb_outputs)
+end
+
+# linear equality constraints
+A = [1.0 1.0 1.0 1.0 1.0;
+     0.0 0.0 1.0 -2.0 -2.0]
+b = [5.0; -3.0]
+
+# Define blackbox
+p = NomadProblem(5, 1, ["OBJ"], # the equality constraints are not counted in the outputs of the blackbox
+                 bb,
+                 lower_bound = -10.0 * ones(5),
+                 upper_bound = 10.0 * ones(5),
+                 A = A, b = b)
+
+# Fix some options
+p.options.max_bb_eval = 500
+
+# Define starting points. It must satisfy A x = b.
+x0 = [0.57186958424864897665429452899843;
+      4.9971472653643420613889247761108;
+      -1.3793445664086618762667058035731;
+      1.0403394252630473459930726676248;
+      -0.2300117084673765077695861691609]
+
+# Solution
+result = solve(p, x0)
+println("Solution: ", result.x_best_feas)
+println("Satisfy Ax = b: ", A * result.x_best_feas ≈ b)
+println("And inside bound constraints: ", all(-10.0 .<= result.x_best_feas .<= 10.0))
+```
+
+The reader can take a look at the `test` folder for more difficult examples.
```

Comment:
J'ai l'exemple que ne fonctionne pas lorsque je le teste en local et il y a un problème avec la documentation de `nomadParameters` et `nomadParameters` : 
```julia
julia> include("make.jl")
[ Info: SetupBuildDirectory: setting up build directory.
[ Info: Doctest: running doctests.
[ Info: ExpandTemplates: expanding markdown templates.
┌ Warning: undefined binding 'nomadParameters' in `@docs` block in src/nomadParameters.md:5-7
│ ```@docs
│ nomadParameters
│ ```
└ @ Documenter.Expanders ~/.julia/packages/Documenter/FuXcO/src/Expanders.jl:298
┌ Warning: undefined binding 'nomadResults' in `@docs` block in src/nomadResults.md:5-7
│ ```@docs
│ nomadResults
│ ```
└ @ Documenter.Expanders ~/.julia/packages/Documenter/FuXcO/src/Expanders.jl:298
┌ Warning: failed to run `@example` block in src/tutorial.md:95-133
│ ```@example HS48
│ 
│ # blackbox
│ function bb(x)
│   f = (x[1]- 1)^2 * (x[2] - x[3])^2 + (x[4] - x[5])^2
│   bb_outputs = [f]
│   success = true
│   count_eval = true
│   return (success, count_eval, bb_outputs)
│ end
│ 
│ # linear equality constraints
│ A = [1.0 1.0 1.0 1.0 1.0;
│      0.0 0.0 1.0 -2.0 -2.0]
│ b = [5.0; -3.0]
│ 
│ # Define blackbox
│ p = NomadProblem(5, 1, ["OBJ"], # the equality constraints are not counted in the outputs of the blackbox
│                  bb,
│                  lower_bound = -10.0 * ones(5),
│                  upper_bound = 10.0 * ones(5),
│                  A = A, b = b)
│ 
│ # Fix some options
│ p.options.max_bb_eval = 500
│ 
│ # Define starting points. It must satisfy A x = b.
│ x0 = [0.57186958424864897665429452899843;
│       4.9971472653643420613889247761108;
│       -1.3793445664086618762667058035731;
│       1.0403394252630473459930726676248;
│       -0.2300117084673765077695861691609]
│ 
│ # Solution
│ result = solve(p, x0)
│ println("Solution: ", result.x_best_feas)
│ println("Satisfy Ax = b: ", A * result.x_best_feas ≈ b)
│ println("And inside bound constraints: ", all(-10.0 .<= result.x_best_feas .<= 10.0))
│ ```
│   c.value = UndefVarError: NomadProblem not defined
└ @ Documenter.Expanders ~/.julia/packages/Documenter/FuXcO/src/Expanders.jl:563
[ Info: CrossReferences: building cross-references.
[ Info: CheckDocument: running document checks.
[ Info: Populate: populating indices.
[ Info: RenderDocument: rendering document.
[ Info: HTMLWriter: rendering HTML pages.
```

Reply:
I do not understand. Normally, it should not take into consideration `src/nomadParameters.md` as it is not included in `make.jl`

Reply:
It's `docs/src/nomadParameters.md` that is not included in `make.jl` and not `src/nomadParameters.md`.